.github/workflows/release.yml

```diff
@@ -221,13 +221,6 @@ jobs:
           name: neptune-package
           path: dist/
 
-#      - name: Uploading to test PyPI
```

Comment:
Please leave it :P. I was thinking about restoring it a couple of days ago :wink: